crowbar_framework/config/initializers/tmpdir.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Dir
+  class << self
+    alias_method :o_mktmpdir, :mktmpdir
+
+    def Dir.mktmpdir(prefix_suffix=nil, tmp_dir=nil, *rest)
+      tmp_dir = Rails.root.join("tmp").to_s if tmp_dir.nil?
```

Comment:
Donn't know it's on the card see https://trello.com/c/v8GWRrgw/600-3-s36p9-cleanup-tmpdirs-after-usage.